src/cmd/create.go

```diff
@@ -666,18 +668,19 @@ func pullImage(image, release string) (bool, error) {
 	}
 
 	var imageFull string
+	var imageSize string = "docker://"
```

Comment:
The `docker://` prefix looks like an implementation detail of the `skopeo inspect ...` wrapper.  So, I would suggest moving it there.